libraries/AC_Autorotation/AC_Autorotation.cpp

```diff
@@ -326,35 +326,34 @@ void AC_Autorotation::update_forward_speed_controller(float pilot_accel_norm)
     _limit_accel = bf_accel_target_mss.limit_length(get_accel_max_mss());
 
     // Calculate roll and pitch targets from angles, negative accel for negative pitch (pitch forward)
-    Vector2f angle_target_deg = { accel_mss_to_angle_deg(-bf_accel_target_mss.x), // Pitch
-                              accel_mss_to_angle_deg(bf_accel_target_mss.y)}; // Roll
+    Vector2f angle_target_rad = { accel_mss_to_angle_rad(-bf_accel_target_mss.x), // Pitch
+                                  accel_mss_to_angle_rad(bf_accel_target_mss.y)}; // Roll
 
     // Ensure that the requested angles do not exceed angle max
-    // todo: FIX BUG: degrees limited in centidegrees
-    _limit_accel |= angle_target_deg.limit_length(_attitude_control->lean_angle_max_cd());
+    _limit_accel |= angle_target_rad.limit_length(_attitude_control->lean_angle_max_rad());
```

Comment:
@Ferruccio1984, @MattKear FYI, we're fixing this angle limit here so this is a change in behaviour

Reply:
@rmackay9 thanks for the heads up and for fixing this.  I am happy with the change in behaviour.